tensorboard/webapp/metrics/data_source/BUILD

```diff
@@ -96,3 +109,16 @@ tf_ts_library(
         "@npm//@types/jasmine",
     ],
 )
+
+tf_ts_library(
+    name = "saved_pins_data_source_test",
+    testonly = True,
+    srcs = [
+        "saved_pins_data_source_test.ts",
+    ],
+    deps = [
+        ":saved_pins_data_source",
+        "//tensorboard/webapp/angular:expect_angular_core_testing",
```

Comment:
If you haven't already, please don't forget to use copybara_presubmit to import the changes into a local google3 repo and run TAP on it.

(I have no reason to believe that the import will fail but I just want to make sure you've done that step).

Reply:
Thank you! I almost forgot to use `copybara_presubmit`.  I checked that TAP passed.